tests/test_s3/test_s3_file_handles.py

```diff
@@ -39,10 +45,20 @@ class TestS3FileHandleClosures(TestCase):
     def setUp(self) -> None:
         if settings.TEST_SERVER_MODE:
             raise SkipTest("No point in testing ServerMode, we're not using boto3")
-        self.s3 = s3model.S3Backend("us-west-1", "1234")
```

Comment:
This had to be changed because now we make references to `s3_backends` from with the `S3Backend` class